### PR TITLE
Update defender-for-office-365-whats-new.md

### PR DESCRIPTION
**Pull request by VipulMSFT**
Adding a few AST updates to What's new page.

### CODE REVIEW

File: microsoft-365/security/office-365-security/defender-for-office-365-whats-new.md [hunk 1]
```diff
@@ -52,6 +56,7 @@ For more information on what's new with other Microsoft Defender security produc
...
 - **Configuring third-party phishing simulations in Advanced Delivery:** We've expanded "Simulation URLs to allow" limit to 30 URLs. To learn how to configure, see [Configure the delivery of third-party phishing simulations to users and unfiltered messages to SecOps mailboxes](skip-filtering-phishing-simulations-sec-ops-mailboxes.md)
+- [Enhanced user telemetry in the simulation reports in Attack Simulation Training](https://techcommunity.microsoft.com/t5/microsoft-defender-for-office/attack-simulation-training-new-insights-into-targeted-user/ba-p/3673105): As part of our enhanced user telemetry, administrators can now view additional details on how their targeted users are interacting with the phishing payload from phishing simulation campaigns.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/defender-for-office-365-whats-new.md]:
```suggestion
- [Enhanced user telemetry in the simulation reports in Attack Simulation Training](https://techcommunity.microsoft.com/t5/microsoft-defender-for-office/attack-simulation-training-new-insights-into-targeted-user/ba-p/3673105): As part of our enhanced user telemetry, administrators can now view additional details about how their targeted users are interacting with the phishing payload from phishing simulation campaigns. 
```

File: microsoft-365/security/office-365-security/defender-for-office-365-whats-new.md [hunk 2]
```diff
@@ -42,6 +42,10 @@ For more information on what's new with other Microsoft Defender security produc
 - [What's new in Microsoft Defender for Identity](/defender-for-identity/whats-new)
 - [What's new in Microsoft Defender for Cloud Apps](/cloud-app-security/release-notes)
 
+## April 2023
+
+- [Using machine learning to drive more effective simulations in Attack Simulation and Training](https://techcommunity.microsoft.com/t5/microsoft-defender-for-office/attack-simulation-training-using-machine-learning-to-drive-more/ba-p/3791023): Make use of intellgent PCR (Predicted Compromise Rate) and Microsoft Defender for Office 365 payload recommendations for utilizing high quality payloads in your simulation.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/defender-for-office-365-whats-new.md]:
```suggestion
- [Using machine learning to drive more effective simulations in Attack Simulation and Training](https://techcommunity.microsoft.com/t5/microsoft-defender-for-office/attack-simulation-training-using-machine-learning-to-drive-more/ba-p/3791023): Make use of intelligent predicted compromise rate (PCR) and Microsoft Defender for Office 365 payload recommendations for utilizing high-quality payloads in your simulation. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @VipulMSFT, kindly commit the suggested changes. Many thanks.

**Comment by VipulMSFT** [comment]
@JohanFreelancer9 - updated the PR with proposed changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by VipulMSFT** [comment]
@denisebmsft @MSFTTracyP can you please merge this PR, if everything looks good? Thank you!

**Comment by VipulMSFT** [comment]
@denisebmsft @MSFTTracyP can you please merge this PR, if everything looks good? Thank you!

**Comment by VipulMSFT** [comment]
Hi  @denisebmsft @MSFTTracyP - can you please merge this PR?

**Pull request merged**